### PR TITLE
docs(router): get rid of unnecessary line in wildcard route code example and fixing wildcard docregions

### PR DESCRIPTION
**Pull request by dalejodc**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Other information

The [wildcard example](https://angular.io/guide/router#wildcard-route-how-to) leads to display a 404 page with the `PageNotFoundComponent`. But before, there is a wildcard to redirect to the `FirstComponent` and because of the routes order the `FirstComponent` will be displayed; which it is not the target of the wildcard route example code.

### CODE REVIEW

File: aio/content/examples/router/src/app/app-routing.module.8.ts [hunk 1]
```diff
@@ -8,7 +8,6 @@ const routes: Routes = [
   { path: 'second-component', component: SecondComponent },
   // #enddocregion routes
   { path: '',   redirectTo: '/first-component', pathMatch: 'full' }, // redirect to `first-component`
-  { path: '**', component: FirstComponent },
```
Comment (gkalpak) [inline aio/content/examples/router/src/app/app-routing.module.8.ts:11]: Good catch! Actually, the code snippets (i.e. docregions) are wrong here. this whole `routes` block should look like:

```diff
 // #docregion routes, routes-with-wildcard, redirect
 const routes: Routes = [
   { path: 'first-component', component: FirstComponent },
   { path: 'second-component', component: SecondComponent },
-  // #enddocregion routes
+  // #enddocregion routes, routes-with-wildcard
   { path: '',   redirectTo: '/first-component', pathMatch: 'full' }, // redirect to `first-component`
-  { path: '**', component: FirstComponent },
-  // #enddocregion redirect
+  // #docregion routes-with-wildcard
   { path: '**', component: PageNotFoundComponent },  // Wildcard route for a 404 page
   // #docregion routes
-  // #docregion redirect
 ];
 // #enddocregion routes, routes-with-wildcard, redirect
```

Could you make this change, @dalejodc?

**Review by gkalpak — CHANGES_REQUESTED** [review]

File: aio/content/examples/router/src/app/app-routing.module.8.ts (hunk 1 above)
Comment (dalejodc) [reply aio/content/examples/router/src/app/app-routing.module.8.ts:11]: For sure! I made the requested changes, @gkalpak.

**Comment by Splaktar** [comment]
@dalejodc CircleCI wants you to rebase 😄 

>CircleCI config on master has been modified since commit 42a9e5a,
        which this PR is based on.
> Please rebase the PR on master after fetching from upstream.
> Rebase instructions for PR Author, please run the following commands:
```bash

          git fetch upstream master;
          git checkout docs;
          git rebase upstream/master;
          git push --force-with-lease;
```

**Review by Splaktar — APPROVED** [review]
AIO preview LGTM, thank you for finding and fixing this!

Just need to rebase as I mentioned above.

**Comment by dalejodc** [comment]
Thank you @Splaktar . I rebased upstream master.

**Comment by Splaktar** [comment]
The [CircleCI lint check](https://circleci.com/gh/angular/angular/717176?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link) is now complaining that the first line of your first commit is over 110 characters.
```
The commit message body does not meet the minimum length of 100 characters
```
You should do an interactive rebase and squash the commits, and during the process update the new commit message to not have any lines over 110 chars. Please let us know if you have questions about squashing commits via interactive git rebase.

**Comment by dalejodc** [comment]
Thanks @Splaktar. I did an interactive squash and rebased upstream master again. But I am not sure why CircleCI is still failling in the commit message length. If I am not wrong the commit message only has 67 characters.

**Comment by atscott** [comment]
Hi @dalejodc, thanks again for fixing this issue. The CI failure is actually because the body in the commit message needs to be at least 100 characters. This check helps us ensure that we don't have commits that aren't describing exactly what was changed and why. 

This PR may be a good example of that. The header gives an overview, but it doesn't really describe why that line was unnecessary. In addition, to get to that 100 characters, you could also note the other change you made in response to George's comment.

**Comment by dalejodc** [comment]
Thanks a lot @atscott. I updated the commit description and it seems that it passed the CI test. :raised_hand:

Could it be a good idea to update the [CONTRIBUTING.md](https://github.com/angular/angular/blob/master/CONTRIBUTING.md)?. I think that the first example  (line #179) would not pass the test. If so, I'll be happy to contribute with that.

**Comment by Splaktar** [comment]
Oh wow, thanks @atscott. I had never heard of a check for the commit body being too short. I didn't read that message from CI carefully enough.

@dalejodc yeah, the Contributing guide examples need to be updated to pass this new min body length CI check. We also need to update the commit message guidelines to state this new rule and describe its purpose. Can you please open a new issue for this before you start any work on it (in a separate PR)?

**Review by gkalpak — APPROVED** [review]
Thx, @dalejodc!

**Comment by dalejodc** [comment]
Sure! I will open a new issue for the contributing guide examples. @Splaktar

**Comment by kapunahelewong** [comment]
LGTM, thank you, @dalejodc! I've rebased for you to see if we can get this green and merge it.

**Comment by kapunahelewong** [comment]
Reviewed-for: global-docs-approvers

🙂

**Review by atscott — APPROVED** [review]

**Review by kapunahelewong — APPROVED** [review]
Reviewed-for: global-docs-approvers

**Pull request closed without merging**